src/static/api_documentation.yml

```
@@ -169,6 +187,14 @@ components:
             type: string
             example: Could not find flood model output 17
 
+    MedusaScenarioIDNotFound:
+      description: The scenario with that MedusaScenarioID could not be found
+      content:
+        text/plain:
+          schema:
+            type: string
+            example: Could not find the model output 31
```

```suggestion
            example: Could not find rainfall scenario 31
```
I looked into this closer and it needs to match `app.py`